Project/frontend/src/components/Graph/index_visjs.tsx

```diff
@@ -223,7 +261,32 @@ const GraphVisualization = () => {
         <option value="grid">Grid</option>
         <option value="random">Random</option>
       </select>
-      <VisGraph graphData={graphData} options={options} />
+      <section className="graph_container">
+        <div className="graph_info">
+          <h1>Graph Information</h1>
+          <p>
+            Document Name: <br /> {graphData.document_name}
+            <br /> <br />
+            Created at: <br /> {formattedDate} {formattedTime}
+          </p>
+          <div className="search_container">
+            <input
+              type="text"
+              placeholder="Search for keywords"
+              style={inputStyle}
+              onKeyDown={searchGraph}
+            />
+            <FaSearch className="search_icon" />
+          </div>
+          <textarea
```

Comment:
Same here. Would be nice if you use MUI input component for UX consistency https://mui.com/material-ui/react-text-field/

Reply:
changed to mui text field